unstructured/documents/elements.py

```diff
@@ -35,6 +40,60 @@ def to_dict(self):
         return {key: value for key, value in self.__dict__.items() if value is not None}
 
 
+@dataclass
+class CoordinatesMetadata:
+    """Metadata fields that pertain to the coordinates of the element."""
+
+    points: Tuple[Tuple[float, float], ...]
+    system: CoordinateSystem
```

Comment:
@qued you mentioned that points without a system (and a coordinate system without points) isn't very meaningful. Therefore, I made both the points and system parameters non-optional in this constructor.

I also got rid of the `merge` method I added for `CoordinatesMetadata` to avoid generating `CoordinatesMetadata` instances with `points` and `system` from disparate sources that are incompatible with each other.